tests/show_techsupport/test_techsupport.py

```diff
@@ -136,9 +136,16 @@ def neighbor_ip(duthost, testbed):
     # ptf-32 topo is not supported in mirroring
     if testbed['topo']['name'] == 'ptf32':
         pytest.skip('Unsupported Topology')
-
     mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
-    dst_ip = mg_facts["minigraph_portchannel_interfaces"][0]['peer_addr']
+    dst_ip = None
+    if mg_facts["minigraph_portchannel_interfaces"]:
+        dst_ip = mg_facts["minigraph_portchannel_interfaces"][0]['peer_addr']
+    else:
+        dst_ip = [(item['peer_addr']) for item in mg_facts["minigraph_interfaces"] if 'peer_addr' in item][0]
```

Comment:
the list may be empty so [0] may throw index error.

Reply:
Fixed